e2e/app/definition.go

```diff
@@ -34,11 +35,10 @@ type DefinitionConfig struct {
 	InfraProvider string
 
 	// Secrets (not required for devnet)
-	DeployKeyFile  string
-	RelayerKeyFile string
```

Comment:
removed this flag

Reply:
We need to remove this from the ops repo as well.

Reply:
indeed